wgetpaste

```diff
@@ -657,7 +658,11 @@ showexpirations() {
 }
 
 showurl() {
-	echo -n "Your ${2}paste can be seen here: " >&2
+	if [[ $QUIET ]]; then
+		echo -n "" >&2
+	else
+		echo -n "Your ${2}paste can be seen here: " >&2
+	fi
```

Comment:
Instead of writing an empty string to `stderr` when `-q`, I'd instead test for not `-q` before writing the "Your paste..." string.